app/logic/emailHandler.py

```diff
@@ -168,6 +168,8 @@ def saveAttachment(self):
             self.attachment_file.save(self.getAttachmentFullPath()) # saves attachment in directory
         except AttributeError: # will pass if there is no attachment to save
             pass
+        except IsADirectoryError: # will pass if we try to handle a directory as a file
+            pass
```

Comment:
Seems like we should prevent `getAttachmentFullPath` from returning only a directory path in the first place.

Reply:
resolved